uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
+
+#
+# Heater State
+# 
+uint8 FLAG_HEATER_AVAILABLE      = 0
+uint8 FLAG_HEATER_WORKING        = 1
+uint8 FLAG_HEATER_OVERCURRENT    = 2
+uint8 FLAG_HEATER_OPENCIRCUIT    = 3
+uint8 flags
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 baro_sensor_temperature         # Kelvin
+float16 aspd_sensor_temperature         # Kelvin
```

Comment:
I think that static_sensor_temperature and differential_sensor_temperature is the most precise.